src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/Filter/search_specification.rst

```diff
@@ -520,20 +519,49 @@ IN
 ""
 :Type: filter
 
-.. code-block:: yaml
+.. code-block:: php
 
-    terms:
-        color-option.id: [5, 6, 7]
+    'filter' => [
+        'terms' => [
+            'color-option' => ['red']
+        ]
+    ]
 
 EMPTY
 """""
+:Type: must_not
+
+.. code-block:: php
+
+    'must_not' => [
+        'exists' => [
+            'field' => 'color-option'
+        ]
+    ]
+
+NOT EMPTY
+"""""""""
 :Type: filter
 
-.. code-block:: yaml
+.. code-block:: php
 
-    missing:
-       field: "color-option"
+    'filter' => [
+        'exists' => [
+            'field' => 'color-option'
+        ]
+    ]
+
+NOT IN
+""""""
+:Type: must_not
+
+.. code-block:: php
 
+    'must_not' => [
```

Comment:
Don't know if the leading must_not part is useful as it's precised 3 lines below